scripts/initialize_database.py

```diff
@@ -10,12 +12,11 @@
 )
 import donate.settings as configs
 
-
 def create_session():
     flask_env = os.environ['FLASK_ENV']
     if flask_env == "PRODUCTION":
         config = configs.ProdConfig
-    else:
+    if flask_env == "DEVELOPMENT":
```

Comment:
Probably good to error or warn here if there is no recognized `flask_env` value.